common/pom.xml

```diff
@@ -34,6 +34,11 @@
       <artifactId>hive-classification</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
why this new dependency is required here ?

Reply:
Hi @Indhumathi27,
The issue arose because Hadoop was including the Avro shaded dependency through hadoop-client-api, which in turn was pulled in by orc-core. With JDK 17's reflection changes, Avro's Stringable class became inaccessible. To resolve this, we excluded hadoop-client-api from orc-core which was not required and creating conflict and added Avro explicitly, as we still require this dependency.
You can find more details about the error in the ticket description here: [HIVE-28464](https://issues.apache.org/jira/browse/HIVE-28464).

This change fix the issue